package.json

```diff
@@ -15,5 +15,8 @@
   "license": "ISC",
   "devDependencies": {
     "parcel-bundler": "^1.12.4"
+  },
+  "dependencies": {
+    "chartist": "^0.11.4"
```

Comment:
Since you are bringing Chartist in via CDN in the demo, I don't think it's needed here as a dependency